spec/models/board_spec.rb

```diff
@@ -2,11 +2,47 @@
 require 'matrix'
 
 RSpec.describe Board, type: :model do
-  describe 'new board' do
-    let(:matrix) { Matrix.build(3) { '_'} }
-    context do
-      b = Board.new
+  let(:matrix) { Matrix.build(3) { '_'} }
+  let(:x)      { 0 }
+  let(:y)      { 0 }
+  let(:matrix_marked) { Matrix.rows([['x','_','_'],['_','_','_'],['_','_','_']]).to_a }
+  let(:matrix_full) { Matrix.rows([['x','o','x'],['o','o','x'],['x','o','o']]).to_a }
+  let(:matrix_winning) { Matrix.rows([['x','o','x'],['x','x','o'],['x','o','x']]).to_a }
+
+  describe 'board' do
+    b = Board.new
+    context 'new' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.